libelement/src/object_model/intrinsics/intrinsic_function.hpp

```diff
@@ -20,12 +20,14 @@ namespace element
         bool variadic = false;
     };
 
+    //todo: move somewhere else
```

Comment:
Can you be a bit more detailed here? Why should it be moved? What's an appropriate place?

Reply:
decided to move it :b